src/Platform/Microsoft.Testing.Platform/Logging/FileLoggerProvider.cs

```diff
@@ -16,6 +18,14 @@ internal sealed class FileLoggerProvider : ILoggerProvider, IDisposable
     private readonly string _logPrefixName;
     private readonly bool _customDirectory;
 
+    /* for testing */
+    internal FileLoggerProvider()
+    {
+        _clock = new SystemClock();
+        _logPrefixName = string.Empty;
+        _customDirectory = false;
+    }
```

Comment:
It's not super clear, do you want to avoid the `FileLogger` creation? Otherwise, I don't see why you are not simply providing arguments to the other ctor.

Reply:
Indeed, I wanted to avoid the creation of the `FileLogger`.

Reply:
Reverted.